profiles/arch/base/package.use.mask

```diff
@@ -488,3 +488,7 @@ media-video/mplayer cpudetection
 # Ian Whyman <thev00d00@gentoo.org> (2016-05-22)
 # Mask raspberry-pi except on arm.
 dev-libs/libcec raspberry-pi
+
+# Michal Privoznik <michal.privoznik@gmail.com> (2024-10-10)
```

Comment:
Top of the file (below header), please, not the bottom

Reply:
Fixed. Thanks.